.github/workflows/publish-next-release.yml

```diff
@@ -42,3 +42,40 @@ jobs:
       dist-tag: next-release
     secrets:
       NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+  build-test:
```

Comment:
Thanks for the quick PR! 🎉 

nit: 
1. What do you think if we merge the duplicate here with the [Test and Publish / latest](https://github.com/aws-amplify/amplify-ui/actions/workflows/publish-latest.yml)?
2. Do we need to add metrics?

Reply:
+1, we don't need metrics at this time. We can look at the status on the next-release/main branch